rapidfuzz/details/common.hpp

```diff
@@ -48,8 +48,10 @@ constexpr double result_cutoff(double result, double score_cutoff)
 template <int Max = 1>
 constexpr double norm_distance(int64_t dist, int64_t lensum, double score_cutoff = 0)
 {
-    double max = static_cast<double>(Max);
-    return result_cutoff((lensum > 0) ? (max - max * dist / lensum) : max, score_cutoff);
+    auto max = static_cast<double>(Max);
```

Comment:
When the type is short and completely clear I personally prefer using the type over auto.